examples/timer/Cargo.toml

```
@@ -6,3 +6,4 @@ edition = "2018"
 
 [dependencies]
 yew = { path = "../../yew" }
+js-sys = "0.3.45"
```

Missing newline (and unnecessary patch bound):
```suggestion
js-sys = "0.3"

```